net-im/dingtalk/dingtalk-1.7.0.30424.ebuild

```
@@ -0,0 +1,123 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PGK_NAME="com.alibabainc.dingtalk"
+inherit unpacker xdg
+
+DESCRIPTION="Communication platform that supports video and audio conferencing"
+HOMEPAGE="https://gov.dingtalk.com"
+SRC_URI="https://dtapp-pub.dingtalk.com/dingtalk-desktop/xc_dingtalk_update/linux_deb/Release/com.alibabainc.${PN}_${PV}_amd64.deb"
+
+LICENSE="all-rights-reserved"
+SLOT="0"
+KEYWORDS="-* ~amd64"
+
+RESTRICT="strip mirror bindist"
+
+RDEPEND="
+	dev-libs/libthai
+	dev-qt/qtgui
+	net-nds/openldap
+	media-sound/pulseaudio
+	media-video/rtmpdump
+	net-misc/curl
+	sys-libs/glibc
+	sys-libs/zlib
+	sys-process/procps
+	x11-libs/gtk+:2
+	x11-libs/gtk+:3
+	x11-libs/libXScrnSaver
+"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND="dev-util/patchelf"
+
+S=${WORKDIR}
+
+QA_PREBUILT="*"
+
+src_install() {
+	# Install scalable icon
+	mkdir -p "${S}"/usr/share/icons/hicolor/scalable/apps || die
+	cp "${FILESDIR}"/dingtalk.svg "${S}"/usr/share/icons/hicolor/scalable/apps || die
```

Should inherit desktop to simplify.
```suggestion
	doicon -s scalable "${FILESDIR}"/dingtalk.svg
```